server/test/test_json_test.go

```diff
@@ -0,0 +1,51 @@
+package test_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/kubeshop/tracetest/server/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSpecRetrocompability(t *testing.T) {
```

Comment:
Same for the tests: https://github.com/kubeshop/tracetest/blob/main/server/model/trigger_json_test.go

It's easier to understand the chronological order of the versions